app/static/joke.js

```diff
@@ -0,0 +1,14 @@
+function makejoke() {
+  fetch('/joke')
+    .then(response => response.json())
+    .then(data => alert(data.text));
```

Comment:
Almost perfect!
Now use [SweetAlert2](https://sweetalert2.github.io/) to replace the `alert`. It should be easy and would look much better.

Reply:
Thanks :)
Done

Reply:
Still here

Reply:
Now it is not :)
I checked.